toolkit.py

```diff
@@ -111,6 +116,19 @@ def __get_file_name(self, path):
         name = tail or ntpath.basename(head)
         return name.split('.')[0]
 
+    def check_change(self, frame):
```

Comment:
@Foxiy This is pretty nasty way of doing it (doing disk IO in order to compare to images)...

Could it be perhaps improved?